test/e2e/setup/rosa/setup/teardown.sh

```
@@ -1,46 +1,153 @@
 #!/usr/bin/env bash
 
+######################
+# Teardown Maestro server and AWS resources
+######################
+
+PWD="$(cd "$(dirname ${BASH_SOURCE[0]})" ; pwd -P)"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)